libretroshare/src/services/p3gxscircles.cc

```diff
@@ -1118,7 +1118,7 @@ bool p3GxsCircles::cache_request_load(const RsGxsCircleId &id)
 		return true;
 	}
 
-	int32_t age = 0;
+	uint32_t age = 0;
 	if (RsTickEvent::prev_event_ago(CIRCLE_EVENT_CACHELOAD, age))
```

Comment:
Technically it's not possible to ensure that age will not be negative. That is why this function uses a int32 instead of a uint32_t. So hanging to int32 is probably bad.

Reply:
I'm telling this because, even if the logic ensures that it behaves in some way, a bug can turn this number into a < 0 number and therefore make it way too large. If you can avoid that when it is computed, then it's fine.